css/layout.css

```diff
@@ -29,13 +29,203 @@ body {
   max-width: 18rem;
```

Comment:
- [ ] I see this is a reusable class, which is great for keeping your CSS dry, congratulation! But to achieve the template effect for the line break as the template, this needs to be a smaller breakpoint for that section, you can create a different reusable class or a specific one for this case
![image](https://user-images.githubusercontent.com/73140461/143719613-cf803081-e725-4fa0-af84-b96108bb87d0.png)